### PR TITLE
Reset to base React project

### PR DESCRIPTION
**Pull request by ClayMav**
Using the [Atomic React](https://github.com/diegohaz/arc) boilerplate, which has support off the bat for testing and routing, this is a complete reversal from Nuxt.js to React.js
**Question:**
Should we just nuke the repo and recreate it? Else, accept this PR, and I will go from there.

### CODE REVIEW

**Comment by ClayMav** [comment]
Changed.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: node_js
+node_js:
+  - v6
+script:
+  - npm run lint
```
Comment (kevinschoonover) [inline .travis.yml]: yarn

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  - npm test -- --coverage
```
Comment (kevinschoonover) [inline .travis.yml]: yarn

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  - npm run build
```
Comment (kevinschoonover) [inline .travis.yml]: yarn

File: package.json [hunk 4]
```diff
@@ -1,46 +1,89 @@
...
-    "dev": "nuxt",
-    "build": "nuxt build",
-    "start": "nuxt start",
-    "generate": "nuxt generate",
-    "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
-    "precommit": "yarn run lint",
-    "test": "yarn run unit",
-    "unit": "jest --config test/unit/jest.conf.js --coverage",
-    "watch": "jest --config test/unit/jest.conf.js --watchAll --coverage"
+    "start": "npm run dev",
```
Comment (kevinschoonover) [inline package.json]: yarn + rest of this file

File: src-example/components/molecules/Feature/index.stories.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+      Ullamco duis in labore consectetur ad exercitation esse esse duis mollit sint.
+    </Feature>
+  ))
+  .add('with icon', () => (
+    <Feature icon="close" title="ARc">
+      Ullamco duis in labore consectetur ad exercitation esse esse duis mollit sint.
+    </Feature>
+  ))
+  .add('with code', () => (
+    <Feature code="npm run build" title="ARc">
```
Comment (kevinschoonover) [inline src-example/components/molecules/Feature/index.stories.js]: yarn

File: src-example/components/organisms/FeatureList/index.js [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+        title="Webpack"
+        code="npm run build"
+      >
+        The awesome module bundler with <Link href="https://webpack.github.io/docs/hot-module-replacement.html">Hot Module Replacement</Link> enabled.
+      </StyledFeature>
+      <StyledFeature
+        icon="jest"
+        link="https://facebook.github.io/jest"
+        title="Jest"
+        code="npm run test"
```
Comment (kevinschoonover) [inline src-example/components/organisms/FeatureList/index.js]: yarn

File: src-example/components/molecules/Feature/index.test.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+  expect(wrapper.find({ id: 'foo' })).toHaveLength(1)
+})
+
+it('renders title', () => {
+  const wrapper = wrap()
+  expect(wrapper.contains('Test')).toBe(true)
+})
+
+it('renders code', () => {
+  const wrapper = wrap({ code: 'npm run build' })
```
Comment (kevinschoonover) [inline src-example/components/molecules/Feature/index.test.js]: yarn

**Comment by ClayMav** [comment]
After much time
this is ready to go.

File: .eslintrc [hunk 8]
```diff
@@ -0,0 +1,47 @@
+{
+  "parser": "babel-eslint",
```
Comment (kevinschoonover) [inline .eslintrc]: I would convert this to a .eslintrc.js for future extensibility and make sure we aren't losing some options specified in the old one.

File: .gitignore [hunk 9]
```diff
@@ -1,67 +1,7 @@
-# Logs
```
Comment (kevinschoonover) [inline .gitignore]: I would readd alot of things removed with this .gitignore or look at https://github.com/github/gitignore/blob/master/Node.gitignore

**Pull request merged**